release-notes/dashboards-assistant.release-notes-2.12.0.0.md

```diff
@@ -12,4 +12,5 @@ Compatible with OpenSearch and OpenSearch Dashboards Version 2.12.0
 - Support save conversation to notebook ([3010362](https://github.com/opensearch-project/dashboards-assistant/commit/3010362))([#93](https://github.com/opensearch-project/dashboards-assistant/pull/93))
 
 ### Feature
-- Set verbose to false ([#131](https://github.com/opensearch-project/dashboards-assistant/pull/131))
\ No newline at end of file
+- Set verbose to false ([#131](https://github.com/opensearch-project/dashboards-assistant/pull/131))
+- Fix: comply with the field change of agent framework ([#137](https://github.com/opensearch-project/dashboards-assistant/pull/137))
```

Comment:
Maybe can be added into a new section called `Bug Fixes`, you can refer to https://github.com/opensearch-project/dashboards-observability/blob/main/release-notes/dashboards-observability.release-notes-2.9.0.0.md